special-pages/pages/onboarding/app/components/App2.module.css

```diff
@@ -23,11 +23,11 @@
         transition: transform 0.3s ease-in-out;
         transform: translateY(0);
 
-        &[data-current="welcome"] {
-            transform: translateY(calc(50vh - 288px));
-        }
+        &[data-current="welcome"],
         &[data-current="getStarted"] {
-            transform: translateY(calc(50vh - 288px));
+            /* max() is not supported by stylelint's CSS parser: https://github.com/csstree/csstree/issues/164 */
```

Comment:
Strange that this isn’t supported. I double checked that `max()` is definitely allowed in a `translateY()` by:

- Running it through the [W3 CSS Validator](https://jigsaw.w3.org/css-validator/).
- Testing in Chrome, DuckDuckGo on macOS, and DuckDuckGo on Windows.

Reply:
I feel at this point stylelint's inconsistencies are starting to cause more problems than it solves. I experimented with https://eslint.org/blog/2025/02/eslint-css-support/ when it came out and it was still way behind in feature support, but maybe it's gotten better since.

Reply:
@mgurgel  as discussed we might just drop stylelint in the end, but to move forward, can we bypass this somehow?